bin/ember

```diff
@@ -25,6 +25,40 @@ try {
 
 logger.info('Resolved "ember-cli" to %j', projectLocalCli);
 
+if (!projectLocalCli && !process.versions.pnp) {
```

Comment:
You probably mean `if (!projectLocalCli && process.versions.pnp) {`?

Reply:
no, if `yarn run` is used it will set `process.versions.pnp` which means Yarn PnP mode is already enabled and we don't have to enable it manually. we only need to enable it ourselves if `ember` is executed directly and a `.pnp.js` file is found.